### PR TITLE
user/wgetpaste: new package

### PR DESCRIPTION
**Pull request by jabuxas**

### CODE REVIEW

File: contrib/wgetpaste/template.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+pkgname = "wgetpaste"
+pkgver = "2.34"
+pkgrel = 0
+build_style = ""
```
Comment (nekopsykose) [inline contrib/wgetpaste/template.py:4]: the default is empty if you don't set this at all

**Comment by q66** [comment]
user/

File: contrib/wgetpaste/template.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+pkgname = "wgetpaste"
+pkgver = "2.34"
+pkgrel = 0
+depends = ["bash", "wget2", "chimerautils"]
```
Comment (q66) [inline contrib/wgetpaste/template.py:4]: `depends = ["bash", "wget2"]`

File: user/wgetpaste/template.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+maintainer = "jabuxas <lucasbarbieri.c@gmail.com>"
+license = "MIT"
+url = "https://github.com/zlin/wgetpaste"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "097b77440460365976f8f66e99b0150c8a9527307f6ecade1db6b60a0bfad781"
+
+
+def do_install(self):
+    self.install_bin("wgetpaste")
+    self.install_file("_wgetpaste", "usr/share/zsh/site-functions")
```
Comment (nekopsykose) [inline user/wgetpaste/template.py:15]: install_completion

Comment (nekopsykose) [reply user/wgetpaste/template.py:15]: also put the install_license in here instead of post_install separately

**Pull request merged**